lib/GLPI/Agent/Task/Inventory/Linux/Distro/OSRelease.pm

```diff
@@ -57,6 +61,33 @@ sub _fixDebianOS {
         if $debian_version && $debian_version =~ /^\d/;
 }
 
+sub _fixAstraOS {
+    my (%params) = @_;
+    my $os = $params{os} ||= {};
+
+    if (my $version = getFirstLine(file => '/etc/astra/build_version')) {
+        $os->{VERSION} = $version if $version =~ /^\d/;
+    }
+
+    return unless canRead('/etc/astra_license');
+
+    if (my ($edition) = map { /^DESCRIPTION="?(.*?)"?$/ ? $1 : () }
+                        getAllLines(file => '/etc/astra_license'))
+    {
```

Comment:
Here, you should probably use our `getFirstMatch()` API as this test is not easy to read:
```suggestion
    if (my ($edition) = getFirstMatch(
        pattern => qr/^DESCRIPTION="?(.*?)"?$/,
        file    => '/etc/astra_license'
    )) {
```

Can you test it ?